src/main/java/com/amplitude/HttpCall.java

```diff
@@ -14,10 +13,12 @@
 public class HttpCall {
   private final String apiKey;
   private final String serverUrl;
+  private final Options options;
 
-  protected HttpCall(String apiKey, String serverUrl) {
+  protected HttpCall(String apiKey, String serverUrl, Options options) {
```

Comment:
Since Options are optional can you please add a convenience constructor. This will also make it backwards compatible with previous versions.

```
  protected HttpCall(String apiKey, String serverUrl) {
    this(apiKey, serverUrl, null)
  }
```

Reply:
Done.